main.js

```diff
@@ -10,10 +10,10 @@
  */
 import {CookieJar} from 'tough-cookie';
 import {WebhookClient} from 'discord.js';
-import config from './config.json';
+import config from './config.json' assert { type: 'json' };
```

Comment:
Let's not use these until they become a stable ECMAScript feature, supported by ESLint. Importing JSON modules without assertions works in Node v16, so you can change the `node` engine in `package.json` to `16.x`.

Reply:
It didn't work for me on node 17.9.0 without them

Reply:
Yeah, but Node v16 is the current LTS.

Reply:
Hm it looks like it already is set on package.json?
```json
    "engines": {
        "node": ">=16"
    },
```

Reply:
It is set to `>=16`, implying that Node v17 would work. Since that is inaccurate, it should be changed to `16.x`.

Reply:
alright

Reply:
(You also have to not actually use import assertions to fix LGTM and ESLint errors.)

Reply:
Alright removed, hopefully it still works b/c it didn't work for me without them.